app/assets/stylesheets/application.css.scss

```diff
@@ -187,17 +148,47 @@ body h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
 
 .main_panel {
   padding-left: 150px;
+  .thought-teasers{
+    margin: 10px 0;
+    .thought-headline{
+      background-repeat: no-repeat;
+      background-size: cover;
+      height: 150px;
```

Comment:
Properties should be ordered background-repeat, background-size, box-shadow, height, left, position, width